cmd/backup-manager/app/restore/restore.go

```diff
@@ -130,3 +151,24 @@ func constructBROptions(restore *v1alpha1.Restore) ([]string, error) {
 	args = append(args, config.Options...)
 	return args, nil
 }
+
+// updateProgressAccordingToBrLog update restore progress according to the br log.
+func (ro *Options) updateProgressAccordingToBrLog(line string, restore *v1alpha1.Restore, statusUpdater controller.RestoreConditionUpdaterInterface) {
+	step, progress := backupUtil.ParseRestoreProgress(line)
+	if step != "" {
+		fvalue, progressUpdateErr := strconv.ParseFloat(progress, 64)
+		if progressUpdateErr != nil {
+			klog.Errorf("parse restore %s progress string value %s to float error %v", ro, progress, progressUpdateErr)
+			fvalue = 0
```

Comment:
When occurred error here, we should return, or may cause progress changed from 1-99 to 0.

Reply:
After discussion, i will add check in status update to make sure the progress will be updated is greater than origin progress.